mailbox/api/src/main/java/org/apache/james/mailbox/model/SearchQuery.java

```diff
@@ -80,28 +81,28 @@
 
             /**
              * addr-mailbox of the first "cc" address.
-             * 
+             *
```

Comment:
If you want to remove trailing spaces, do that in a separate commit.

Please also unset your IDE to avoid side effects upon save / commit...